common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -83,6 +85,23 @@ public String toString() {
                 + anchorSection + '}';
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        OverlayPosition that = (OverlayPosition) o;
```

Comment:
I think there is a helper class in one of the google libraries we are including that create a general equals by comparing the fields of the class. Don't remember its name, but I think it's better to use it, otherwise it's easy to like add a field and miss updating equals etc. (If you can't find it, look in Legacy where I think it is used here and there, or let me know and I'll go look it up.)

Reply:
MoreObjects